src/routes/variavel.routes.js

```diff
@@ -5,5 +5,7 @@ const variavelRouter = express.Router();
 const VariavelControler = require('../controller/VariavelController');
 
 variavelRouter.post("/", VariavelControler.createVariavel);
+variavelRouter.post("/2", VariavelControler.requestVariavel);
+variavelRouter.post("/3", VariavelControler.requestVariavelFiltered);
```

Comment:
Mudar rota para algo mais legivel ao utilizador da mesma.
```suggestion
variavelRouter.post("/filteredVariable", VariavelControler.requestVariavelFiltered);
```